docs/netdata-agent/configuration/optimize-the-netdata-agents-performance.md

```diff
@@ -1,68 +1,71 @@
 
-| Optimization                                                                                                                      | CPU                | RAM                | Disk IO            |
-|-----------------------------------------------------------------------------------------------------------------------------------|--------------------|--------------------|--------------------|
-| [Use Centralization Points](#use-centralization-points)                                                                           | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| [Disable Plugins or Collectors](#disable-plugins-or-collectors)                                                 | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| [Reduce data collection frequency](#reduce-data-collection-frequency)                                                                  | :heavy_check_mark: |                    | :heavy_check_mark: |
-| [Lower memory usage for metric retention](#lower-memory-usage-for-metric-retention) |                    | :heavy_check_mark: | :heavy_check_mark: |
-| [Change Database mode](#change-database-mode)                                                                |                    | :heavy_check_mark: | :heavy_check_mark: |
-| [Disable ML on Children](#disable-machine-learning-on-children)                                                                   | :heavy_check_mark: |                    |                    |
-| [Use a reverse proxy](#run-netdata-behind-a-proxy)                                                                                | :heavy_check_mark: |                    |                    |
+| Optimization                                                              | CPU                | RAM                | Disk IO            |
+|---------------------------------------------------------------------------|--------------------|--------------------|--------------------|
+| [Implement Centralization Points](#implement-centralization-points)       | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| [Disable Plugins or Collectors](#disable-plugins-or-collectors)           | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| [Adjust data collection frequency](#adjust-data-collection-frequency)     | :heavy_check_mark: |                    | :heavy_check_mark: |
+| [Optimize metric retention settings](#optimize-metric-retention-settings) |                    | :heavy_check_mark: | :heavy_check_mark: |
+| [Select appropriate Database Mode](#select-appropriate-database-mode)     |                    | :heavy_check_mark: | :heavy_check_mark: |
+| [Disable ML on Children](#disable-machine-learning-on-children)           | :heavy_check_mark: |                    |                    |
 
-## Use Centralization Points
+## Implement Centralization Points
```

Comment:
@ilyam8 cases like these, where you changed the heading, links to the old heading are broken (if any)

I normally do a repo-wide search for otimize-the-netdata-agents-performance.md# and see what comes up with headings

Reply:
I think it would be strange to provide links to these sections, since they contain nothing but links to other sections with relevant information.

Reply:
asking because if there are any, now they are broken